devlab/tests/config.py

```diff
@@ -1,3 +1,5 @@
+img_url = 'http://download.cirros-cloud.net/0.3.3/cirros-0.3.3-x86_64-disk.img'
```

Comment:
Do you need a real image for testing migration? As I know you can touch an empty file and use it for glance. Or just use raw data in API. Please, see https://github.com/MirantisWorkloadMobility/CloudFerry/pull/383/files for reference.